presto-test-jdbc-compatibility-old-server/src/test/java/io/prestosql/TestJdbcResultSetCompatibilityOldServer.java

```diff
@@ -43,7 +43,6 @@
 {
     private static final int NUMBER_OF_TESTED_VERSIONS = 5;
     private static final int TESTED_VERSIONS_GRANULARITY = 3;
-    private static final String TESTED_VERSIONS_ENV_KEY = "TESTED_PRESTO_SERVER_VERSIONS";
```

Comment:
Hmmm. It was still useful to be able to test agains specific versions manually. Maybe not enough to keep it... IDK